modules/apps/static/portal-osgi-web/portal-osgi-web-servlet-jsp-compiler/src/main/java/com/liferay/portal/osgi/web/servlet/jsp/compiler/internal/CompilerWrapper.java

```diff
@@ -333,7 +353,7 @@ private void _collectTLDMappings(
 			}
 
 			_populateTldMappings(
-				StringPool.SLASH.concat(resourcePath), taglibXmls,
+				StringPool.SLASH.concat(resourcePath), bundleUrl, taglibXmls,
```

Comment:
@dantewang After a second though, I think the root cause is this "StringPool.SLASH.concat(resourcePath)" is passed in as "absoluteResourcePath", but it is actually not an absolute path.

Reply:
@tinatian Hmm I don't remember why we provide absolute path in the first place.. I'll try this...